editor/sidebar/last-revision/index.js

```diff
@@ -57,7 +57,8 @@ class LastRevision extends Component {
 		}
 		this.setState( { loading: true } );
 		const postIdToLoad = this.props.postId;
-		this.fetchRevisionsRequest = new wp.api.collections.PostRevisions( {}, { parent: postIdToLoad } ).fetch()
+		const Collection = wp.api.getPostTypeRevisionsCollection( this.props.postType || 'post' );
```

Comment:
Maybe it should be the responsibility of the (or a new) selector to perform the fallback logic, if we expect it to be a common need? i.e. `getCurrentPostType = ( state ) => getCurrentPost( state ).type || 'post';`